test/bson_older_versions_tests.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
```

Comment:
I think a comment block somewhere at the top of this file would be useful to future maintainers, explaining what it is doing and why it's needed.